config/initializers/health_monitor.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+Rails.application.config.after_initialize do
+  HealthMonitor.configure do |config|
+    # config.cache
+
+    config.add_custom_provider(SolrStatus)
```

Comment:
Should change to something like https://github.com/pulibrary/pdc_discovery/blob/main/config/initializers/health_monitor.rb#L6-L9. Sorry if this is a silly question, but does oawaiver utilize solr?

Reply:
I apologize, yes, this uses Solr for the names of authors for publications (https://rubygems.org/gems/sunspot is used in place of Blacklight)